### PR TITLE
[7.x] Update docblock and parameter name to match parent

### PR DESCRIPTION
**Pull request by jochensengier**

### CODE REVIEW

**Comment by taylorotwell** [comment]
I think it's fine as is.

**Comment by jochensengier** [comment]
I understand and agree why you don't want to change the parameter name, but what about the incorrect '@throws' in the docblock?

**Comment by driesvints** [comment]
@jochensengier fixed

File: app/Exceptions/Handler.php [hunk 1]
```diff
@@ -29,27 +29,27 @@ class Handler extends ExceptionHandler
     /**
      * Report or log an exception.
      *
-     * @param  \Throwable  $exception
+     * @param  \Throwable  $e
      * @return void
      *
-     * @throws \Exception
+     * @throws \Throwable
```
Comment (GrahamCampbell) [inline app/Exceptions/Handler.php:35]: ?

Comment (jochensengier) [reply app/Exceptions/Handler.php:35]: If your questioning the change, please see https://github.com/laravel/framework/pull/33888

**Pull request closed without merging**